docs/sql_reference/src/asciidoc/_chapters/sql_clauses.adoc

```diff
@@ -1053,8 +1053,7 @@ expressions in each item must be the same as the number of value columns
 in the column list.
 +
 In the example TRANSPOSE A,B,C AS V, the items are A,B, and C, and the
-value column is V. This form can be thought of as a shorter way of writing TRANSPOSE
-(A),(B),(C) AS (V).
+value column is V. This form can be thought of as a shorter way of writing TRANSPOSE (A),(B),\(C) AS (V).

```

Comment:
Not sure why the slash is before "(C)"; I don't think it is correct. Or is the slash an escape? Maybe "(C)" is interpreted as the copyright symbol by the rendering software?

Reply:
Yes, the slash is an escape, (C) is rendered as © 😁 

![c](https://user-images.githubusercontent.com/20532956/43811315-71eaecde-9aee-11e8-8a1c-56e9875c43da.jpg)

![escape](https://user-images.githubusercontent.com/20532956/43811356-ae7f5482-9aee-11e8-9bc7-ba5523a51471.jpg)